tests/testthat/test-read-data.R

```diff
@@ -57,3 +57,8 @@ test_that("can read spectrum region code from geojson", {
   region_codes <- read_geojson_spectrum_region_codes(shape)
   expect_equal(region_codes, c(10, 11, 12, 13, 14, 15, 16, 17, 18, 19))
 })
+
+test_that("read csv removes empty rows", {
+  data <- read_csv(file.path("testdata", "population_empty_rows.csv"))
+  expect_equal(nrow(data), 2448)
```

Comment:
Perhaps add a smaller test file :)

You could just write one though (untested):

```
path <- tempfile()
writeLines(c("a,b", "1,2", "3,4", "5,6", ",", ","), path)
expect_equal(nrow(read_csv(path)), 3)
```

Reply:
Way better - thanks.